SyncApp/Controllers/ConfigrationsController.cs

```diff
@@ -95,6 +98,29 @@ public async Task<IActionResult> Edit(Configrations configrations)
             }
             return View(configrations);
         }
+ 
+        public async Task<decimal?> GetTaxFromShopify(long id)
+        {
+            try
+            { 
+                var countryService = new CountryService(StoreUrl, ApiSecret);
+                Country country = await countryService.GetAsync(id);
+                var tax = country.Tax;
+                var countryDB = _countriesLogic.GetCountry(id);
+                if (countryDB != null)
+                {
+                    countryDB.CountryTax = tax;
+                    _context.Update(countryDB); 
+                    await _context.SaveChangesAsync();     
+                }
+                return tax;
```

Comment:
Tax in our DB is rounded for two decimal points, when the user will click on GetTax this way will see more than two decimals but when refresh the page it will be rounded, if we can make it consistent would be better. (example: Taiwan country return 0.1676 but when refresh it will be 0.17)

Reply:
Done